sample_files/.travis.yml

```
@@ -1,10 +1,10 @@
 language: python
-sudo: false
 cache: pip
```

@sbidoul 

I saw on https://github.com/OCA/queue/pull/175 that @guewen added cache for pre-commit
You also added it here https://github.com/OCA/storage/pull/12/files#diff-354f30a63fb0907d4ad57269548329e3R6

```suggestion
cache:
  directories:
  - $HOME/.cache/pip
  # Add cache for pre-commit on repo that uses it (mandatory for >= 13)
  - $HOME/.cache/pre-commit
```

Seems a good idea to cache pre-commit hooks downloads. Any reason not to include it here?

---

> Any reason not to include it here?

The only reason is that I forgot to do it :) Thanks for noticing.